### ISSUE TITLE
Illuminate\Broadcasting\BroadcastException after few days

### ISSUE THREAD

**Issue by yurik94** [beyondcode/laravel-websockets#375]
After a week of seamless work today our laravel-websockets implementation is again stalled and it happens often, the result is that a lot of Laravel events fail to broadcast.

supervisorctl output:
`laravel-websocket                RUNNING   pid 61767, uptime 7 days, 12:26:51`

Exception thrown from horizon dashboard:

`Illuminate\Broadcasting\BroadcastException in /var/www/vhosts/brain.alfonsino.delivery/httpdocs/vendor/laravel/framework/src/Illuminate/Broadcasting/Broadcasters/PusherBroadcaster.php:119`

```
/**
     * Broadcast the given event.
     *
     * @param  array  $channels
     * @param  string  $event
     * @param  array  $payload
     * @return void
     */
    public function broadcast(array $channels, $event, array $payload = [])
    {
        $socket = Arr::pull($payload, 'socket');

        $response = $this->pusher->trigger(
            $this->formatChannels($channels), $event, $payload, $socket, true
        );
		
	\Illuminate\Support\Facades\Log::critical('broadcast', [$response, $event, $payload, $socket]);

        if ((is_array($response) && $response['status'] >= 200 && $response['status'] <= 299)
            || $response === true) {
            return;
        }

        throw new BroadcastException(
            is_bool($response) ? 'Failed to connect to Pusher.' : $response['body']
        );
    }
```
Note the Log::critical inserted, its output when a job fail is this:
`production.CRITICAL: broadcast [{"body":false,"status":0},"App\\Events\EventRedacted",{...},null]`

and here's a Sentry log
![image](https://user-images.githubusercontent.com/13189646/81940854-55ac3a80-95f8-11ea-9484-9cea5b5e7981.png)

Our broadcasting.php config:
 ```
'connections' => [

        'pusher' => [
            'driver' => 'pusher',
            'key' => env('PUSHER_APP_KEY'),
            'secret' => env('PUSHER_APP_SECRET'),
            'app_id' => env('PUSHER_APP_ID'),
            'options' => [
                'cluster' => env('PUSHER_APP_CLUSTER'),
                'encrypted' => true,
                'host' => '127.0.0.1',
                'port' => 6001,
                'scheme' => 'https',
                'curl_options' => [
                    CURLOPT_SSL_VERIFYHOST => 0,
                    CURLOPT_SSL_VERIFYPEER => 0
                ]
            ],
            'log' => true
        ],
```

Websocket panel:
![image](https://user-images.githubusercontent.com/13189646/81941930-c1db6e00-95f9-11ea-90df-c720c64b16a1.png)

Any help would be really appreciated, I can give more context and logging if needed.

**Comment by eslamismail** [comment]
replace   pusher config in broadcasting.php with this  
'pusher' => [
            'driver' => 'pusher',
            'key' => env('PUSHER_APP_KEY'),
            'secret' => env('PUSHER_APP_SECRET'),
            'app_id' => env('PUSHER_APP_ID'),
            'options' => [
                'cluster' => env('PUSHER_APP_CLUSTER'),
                'useTLS' => true,
                'host' => '127.0.0.1',
                'port' => 6001,
                'scheme' => 'http',
                'curl_options' => [
                    CURLOPT_SSL_VERIFYHOST => 0,
                    CURLOPT_SSL_VERIFYPEER => 0,
                ],
            ],
        ],

**Comment by carlmagumpara** [comment]
any updates?

**Comment by rennokki** [comment]
It seems to me like it's related to the pusher package, `4.1.4`: https://github.com/pusher/pusher-http-php/compare/v4.1.3...v4.1.4

I have tried to fix it by forcing `4.1.3`: 

```bash
$ composer require pusher/pusher-php-server:"4.1.3"
```

**Comment by raymiec** [comment]
> It seems to me like it's related to the pusher package, `4.1.4`: [pusher/pusher-http-php@v4.1.3...v4.1.4](https://github.com/pusher/pusher-http-php/compare/v4.1.3...v4.1.4)
> 
> I have tried to fix it by forcing `4.1.3`:
> 
> ```shell
> $ composer require pusher/pusher-php-server:"4.1.3"
> ```

This worked for me.

**Comment by rennokki** [comment]
@raymiec @yurik94 @carlmagumpara I can enforce `4.1.3` on the package if that helps out.
